pkg/logs/config/config_test.go

```diff
@@ -606,32 +606,32 @@ func (suite *ConfigTestSuite) TestBuildEndpointsWithVectorHttpsOverride() {
 
 func (suite *ConfigTestSuite) TestBuildEndpointsWithVectorHostAndPortOverride() {
 	suite.config.Set("api_key", "123")
-	suite.config.Set("vector.logs.enabled", true)
-	suite.config.Set("vector.logs.url", "vector.host:8443")
+	suite.config.Set("observability_pipelines_worker.logs.enabled", true)
+	suite.config.Set("observability_pipelines_worker.logs.url", "observability_pipelines_vector.host:8443")
```

Comment:
Is there a reason for using `observability_pipelines_vector` instead of `observability_pipelines_worker` to be coherent with the rest of the PR ?

Reply:
Oops that was a typo. Thanks.